BaseLib/CMakeLists.txt

```diff
@@ -15,6 +26,13 @@ if(MSVC)
 	target_link_libraries(BaseLib INTERFACE WinMM) # needed for timeGetTime
 endif()
 
+if(QT4_FOUND)
+	target_link_libraries(BaseLib PUBLIC Qt4::QtXml Qt4::QtXmlPatterns)
+	if(WIN32 AND CMAKE_CROSSCOMPILING AND OPENSSL_FOUND)
+		target_link_libraries(BaseLib PUBLIC Qt4::QtNetwork ${OPENSSL_LIBRARIES} ws2_32)
```

Comment:
The explicit `PUBLIC` and `INTERFACE` keywords in all of the `target_link_libraries` are removed now. The default behaviour (transitive) is sufficient, no need for finer grained dependencies for now.